src/components/LoginButton.jsx

```diff
@@ -22,14 +34,27 @@ export default function LoginButton() {
 
   const signUserOut = () => firebase.auth().signOut();
   const showLoginDialog = () => setLoginDialogOpen(true);
-  const onClick = isSignedIn ? signUserOut : showLoginDialog;
 
-  return (
-    <>
-      <Button variant="contained" color="primary" onClick={onClick}>
-        {isSignedIn ? 'Sign out' : 'Login'}
-      </Button>
-      <LoginDialog open={loginDialogOpen} onClose={handleClose} />
-    </>
-  );
+  function ProfileDropdown(loginStatus) {
+    if (loginStatus === true) {
+      return (
+        <ProfileMenu
+          signOut={signUserOut}
+          name={userName}
+          email={userEmail}
+          photo={photoUrl}
+        />
+      );
+    }
+    return (
+      <>
+        <LoginDialog open={loginDialogOpen} onClose={handleClose} />
+        <Button variant="contained" color="primary" onClick={showLoginDialog}>
+          Login
+        </Button>
+      </>
+    );
+  }
+
+  return <>{ProfileDropdown(isSignedIn)}</>;
```

Comment:
See [this section](https://en.reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator) in [Conditional Rendering docs](https://en.reactjs.org/docs/conditional-rendering.html) to see how you can simplify this.

Reply:
Alright i'll take a look and include those changes on my next commit

Reply:
Alright this change is done If I did it right